src/zenml/models/flavor_models.py

```diff
@@ -49,14 +88,17 @@ class FlavorBaseModel(BaseModel):
         title="The name of the integration that the Flavor belongs to.",
         max_length=STR_FIELD_MAX_LENGTH,
     )
+    logo_url: str = Field(default="https://tinyurl.com/m4xab3yj")
```

Comment:
Will this tinyurl expire at some point? It seems like it takes me to https://github.com/zenml-io. Is that the intended behavior?

Reply:
will be replaced before merge once we decide on a default logo